src/measure_theory/set_integral.lean

```diff
@@ -284,6 +284,92 @@ begin
   exact (lintegral_mono_set' hst),
 end
 
+
+section continuous_set_integral
+/-! ### Continuity of the set integral
+
+We prove that for any set `s`, the function `λ f : α →₁[μ] E, ∫ x in s, f x ∂μ` is continuous. -/
+
+variables [normed_group E] [measurable_space E] [second_countable_topology E] [borel_space E]
+  {𝕂 : Type*} [is_R_or_C 𝕂] [measurable_space 𝕂]
+  [normed_group F] [measurable_space F] [second_countable_topology F] [borel_space F]
+  [normed_space 𝕂 F]
+  {p : ℝ≥0∞} {μ : measure α}
+
+lemma Lp_to_Lp_restrict_add (f g : Lp E p μ) (s : set α) :
```

Comment:
can you add a docstring for this one? It took me some time to parse it, so let's spare it to other readers.

Reply:
Docstrings added for all Lp_restrict lemmas.